ui/src/pages/QuickSetup/AddPartiesPage.tsx

```diff
@@ -29,6 +56,50 @@ const AddPartiesPage = (props: { localOperator: string; onComplete: () => void }
     </label>
   );
 
+  if (loadingStatus === LoadingStatus.CREATING_ADMIN_CONTRACTS) {
```

Comment:
I moved the logic for loading admin contracts and creating verified identities to this file from the quick setup file because this loading state happens under the `/add-parties` url